app/policies/category_policy.rb

```diff
@@ -0,0 +1,29 @@
+class CategoryPolicy < ApplicationPolicy
+  def index?
```

Comment:
it looks like an unnecessary policy because only the admin has access to it and admin namespace has restrictions